### PR TITLE
#3396 can the page of span info be expanded manually

### PR DESCRIPTION
**Pull request by x22x22**
## resolve issue:
* https://github.com/apache/skywalking/issues/3396

### CODE REVIEW

**Comment by zhaoxunyong** [comment]
I have another opinion:  Why not add an arrow on the top right corner? like here:

Snip20190904_8

This way I can horizontally toggle the entire window rather than the exception window.

**Comment by zhaoxunyong** [comment]
> I have another opinion: Why not add an arrow on the top right corner? like here:
> 
> Snip20190904_8
> 
> This way I can horizontally toggle the entire window rather than the exception window.

This effect you've implemented is ok as well. Great job!

**Comment by zhaoxunyong** [comment]
This effect you've implemented is ok as well. Great job!

File: src/assets/svg/magnify.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg t="1567575641976" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="3448" xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="200"><defs><style type="text/css"></style></defs><path d="M626.41 442.15H511.24V327a38.39 38.39 0 1 0-76.78 0v115.15H319.3a38.39 38.39 0 0 0 0 76.78h115.16v96a38.39 38.39 0 1 0 76.78 0v-96h115.17a38.39 38.39 0 1 0 0-76.78z" p-id="3449"></path><path d="M900.75 846.47L753.61 699.33a359.13 359.13 0 0 0 80.11-226.48c0-199-161.88-360.85-360.87-360.85S112 273.88 112 472.85s161.88 360.87 360.85 360.87a359.13 359.13 0 0 0 226.48-80.11l147.14 147.14a38.38 38.38 0 0 0 54.28-54.28z m-712-373.62c0-156.63 127.44-284.07 284.07-284.07s284.12 127.44 284.12 284.07-127.42 284.09-284.09 284.09-284.07-127.42-284.07-284.09z" p-id="3450"></path></svg>
```
Comment (kezhenxu94) [inline src/assets/svg/magnify.svg]: Need license header here

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "vue": "^2.6.6",
     "vue-class-component": "^6.0.0",
     "vue-i18n": "^8.10.0",
+    "vue-js-modal": "^1.3.31",
```
Comment (kezhenxu94) [inline package.json]: Remember to update the LICENSE for the newly-added dependency

File: package-lock.json [hunk 3]
```diff
@@ -19,20 +19,20 @@
...
-        "@babel/traverse": "7.3.4",
-        "@babel/types": "7.3.4",
-        "convert-source-map": "1.6.0",
-        "debug": "4.1.1",
-        "json5": "2.1.0",
-        "lodash": "4.17.11",
-        "resolve": "1.10.0",
-        "semver": "5.6.0",
-        "source-map": "0.5.7"
+        "@babel/code-frame": "^7.0.0",
```
Comment (kezhenxu94) [inline package-lock.json]: If I remember correctly, `^7.0.0` means any version newer than `7.0.0` (or equals to), and `npm install` will usually install the latest version (in a new machine that did not run `npm install` before), which may cause problems when the versions are not compatible, I have encountered this kind of problem before so I recommend to keep these unchanged, WDYT? @TinyAllen Did I miss anything?

File: src/views/components/trace/trace-detail-chart-tree.vue [hunk 4]
```diff
@@ -263,4 +290,8 @@ export default {
      fill: rgba(0,0,0,0.05)
    }
  }
+.dialog-c-text {
```
Comment (kezhenxu94) [inline src/views/components/trace/trace-detail-chart-tree.vue]: Since the contents are exceptions, it would be more comfortable if the font is monospaced.

**Comment by x22x22** [comment]
@wu-sheng 
vue-js-modal not more libs imported.

File: src/assets/svg/magnify.svg (hunk 1 above)
Comment (x22x22) [reply src/assets/svg/magnify.svg]: It has been modified

File: src/views/components/trace/trace-detail-chart-tree.vue (hunk 4 above)
Comment (x22x22) [reply src/views/components/trace/trace-detail-chart-tree.vue]: It has been modified

File: package-lock.json (hunk 3 above)
Comment (x22x22) [reply package-lock.json]: After executing the command 'npm i vue-js-modal', the 'package-lock.json' file is automatically modified. I don't know why it is automatically added with '^'.

**Comment by kezhenxu94** [comment]
> UI license must update, what license of this new dependency? Is there more libs imported because this lib depends?

@wu-sheng I saw it's under MIT license

@x22x22 AFAIK, many npm package uses other modules directly or indirectly, please make sure that all of the transitive dependencies (imported by vue-js-modal that you don't import directly) are allowed in Apache project, and update them(if any)) in the LICENSE file https://github.com/apache/skywalking/tree/master/apm-dist/release-docs/licenses/ui-licenses

File: package.json (hunk 2 above)
Comment (kezhenxu94) [reply package.json]: > Add it here?
> https://github.com/apache/skywalking/blob/master/apm-dist/release-docs/LICENSE

It's for backend dependencies I think, frontend dependencies should be added here https://github.com/apache/skywalking/tree/master/apm-dist/release-docs/licenses/ui-licenses

Comment (x22x22) [reply package.json]: done

**Comment by x22x22** [comment]
> > UI license must update, what license of this new dependency? Is there more libs imported because this lib depends?
> 
> @wu-sheng I saw it's under MIT license
> 
> @x22x22 AFAIK, many npm package uses other modules directly or indirectly, please make sure that all of the transitive dependencies (imported by vue-js-modal that you don't import directly) are allowed in Apache project, and update them(if any)) in the LICENSE file https://github.com/apache/skywalking/tree/master/apm-dist/release-docs/licenses/ui-licenses

done

File: package-lock.json (hunk 3 above)
Comment (kezhenxu94) [reply package-lock.json]: @TinyAllen any idea? Is this a matter? If not others look good to me

Comment (x22x22) [reply package-lock.json]: @TinyAllen 
Or you tell me what parameters to add, npm can automatically add 'vue-js-moda' to the 'package-lock.json' file, without automatically adding the '^' symbol.

Comment (x22x22) [reply package-lock.json]: @kezhenxu94 
Don't worry, I changed back.

**Pull request merged**